### ISSUE TITLE
Feature request: Cancel sync button

### ISSUE THREAD

**Issue by masics** [icetee/remote-ftp#313]
It would be a very handy feature to have a "cancel sync" button somewhere. Some syncs are very long and sometimes I want to cancel them and try something else.

**Comment by nemoyatpeace** [comment]
Yes, this would be nice.  And also a way of knowing the progress of the sync.  I started a sync, but can't tell if it is progressing or not.  Lots of images in one of the folders that changed.
